contracts-periphery/Makefile

```diff
@@ -0,0 +1,39 @@
+# include .env file and export its env vars
+# (-include to ignore error if it does not exist)
+-include .env
+
+all: clean remove install update solc build 
+
+# Install proper solc version.
+solc:; nix-env -f https://github.com/dapphub/dapptools/archive/master.tar.gz -iA solc-static-versions.solc_0_8_10
+
+# Clean the repo
+clean  :; forge clean
+
+# Test your contract on a forked network
+test   :; forge test -f ${ETH_RPC_URL} --fork-block-number 10465092
```

Comment:
We should add a similar snapshot command, then add an `UmbraBatchSendGas` contract which does all required setup in `setUp` and has one-line tests that only send various combinations of tokens and ETH. This way we can use it as a benchmark for optimizations in a future PR

```
snapshot   :; forge snapshot -f ${ETH_RPC_URL} --fork-block-number 10465092
```